spec/models/user_spec.rb

```diff
@@ -1,5 +1,27 @@
 require "rails_helper"
 
 RSpec.describe User, type: :model do
```

Comment:
Seems like, we can remove redundant code
```suggestion
describe User do
```